app/feature/feature-forever/src/main/kotlin/com/hedvig/android/feature/forever/ForeverViewModel.kt

```diff
@@ -93,7 +94,7 @@ class ReferralsViewModel(
   }
 }
 
-data class ReferralsUiState(
+data class ForeverUiState(
```

Comment:
Also probably can be `internal`.
Likely that almost everything in the entire module can be internal now, aside from the koin module and the foreverGraph().
Maybe a thing or two more too, but those for sure.
Doesn't have to be part of this PR necessarily, we can alway do it as we see those things in the future.